hilfe/handbuch/mitarbeiter/arbeitszeitkonto/index.md

```diff
@@ -85,14 +85,13 @@ Beschäftigungsverhältnis saldierend fort. Das Arbeitszeitkonto eines Jahres wi
 immer automatisch in das Folgejahr übertragen.
 
 In vielen Fällen ist es allerdings notwendig, korrigierend in die Saldierung von
-Konten einzugreifen, zum Beispiel wenn der Resturlaub eines Mitarbeiters am Ende
-des Jahres verfallen soll oder die Konten eines Mitarbeiters nach langer
+Konten einzugreifen, zum Beispiel wenn die Konten eines Mitarbeiters nach langer
 Abwesenheit zurückgesetzt werden sollen.
 
 Pentacode bietet Ihnen hierfür die Möglichkeit, den Übertrag eines Monats für
 eines oder alle Konten eines Mitarbeiters zu bearbeiten.
 
-### Kontenübertrag Eines Einzelnen Mitarbeiters Bearbeiten
+### Kontenübertrag eines einzelnen Mitarbeiters Bearbeiten
```

Comment:
das hier ist eigentlich Title-Case, also in Überschriften bewusst jedes Wort groß

gilt auch für weitere Änderungen in diesem PR